tests/integration/models/test_preferences.py

```diff
@@ -49,10 +49,10 @@ def test_update(self, _):
         int_params = ('min_comment_score', 'min_link_score', 'num_comments',
                       'numsites')
         # parameters that accept string types, and two valid values
-        str_params = {'default_comment_sort': ('confidence', 'top'),
-                      'lang': ('es', 'en'),
-                      'media': ('on', 'off'),
-                      'media_preview': ('on', 'off')}
+        str_params = (('default_comment_sort', ('confidence', 'top')),
+                      ('lang', ('es', 'en')),
+                      ('media', ('on', 'off')),
+                      ('media_preview', ('on', 'off')))
```

Comment:
Turns out dicts are ordered in CPython 3.6, which explains why this issue didn't jump out initially to me.